src/cli/gui.py

```diff
@@ -0,0 +1,63 @@
+"""
+Start the GUI. A small tray icon will be displayed. Right click on that to
+access instantshare's functionality.
+
+The GUI is also required for native hotkey support (which uses the hotkeys
+set in instantshare.conf). You can most likely also directly map hotkeys
+to commands using your operating system or desktop environment settings.
+
+Usage:  instantshare gui [options]
+
+Options:
+  -h, --help  Display this information
+"""
+from docopt import docopt
+from cli.main import execute_command
+
+import logging
+
+from tools.config import CONFIG
+
+
+def main(argv):
+    args = docopt(__doc__, argv)
+    from gui.traymenu import Tray
+    from tools.hotkey import Hotkey, HotkeyInUseError, InvalidHotkeyError
+    from tools.toolbox import delay_execution
+    from queue import Queue
```

Comment:
Is there a reason to import this in the main function instead of the top of the module?
Would probably move this to Line 16.